docs/apis/streaming/connectors/rabbitmq.md

```diff
@@ -71,23 +71,25 @@ Example:
 <div class="codetabs" markdown="1">
 <div data-lang="java" markdown="1">
 {% highlight java %}
+RMQConnectionConfig connectionConfig = new RMQConnectionConfig.Builder().build();
```

Comment:
As I've written below, I would prefer if users would not rely on the fact that "localhost" is the default value here.
Once the builder has been changed, we would expect the user to call `.setHost("localhost")` here.